README.md

```diff
@@ -67,6 +67,7 @@ The plugin integrates card component (Secured Fields) using Adyen Checkout for a
  - Vipps
  - WeChat Pay
  - Open Banking / Pay by Bank
+ - Giftcard
```

Comment:
There is already a line for Giftcards. Could you please remove the redundant line?